extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.js

```diff
@@ -82,22 +74,28 @@ export class AmpStoryDevToolsTabLogs extends AMP.BaseElement {
 
     /** @private  {string} */
     this.storyUrl_ = '';
+
+    /** @private {!Array} */
+    this.errorList_ = [];
   }
 
   /** @override */
   buildCallback() {
     this.storyUrl_ = this.element.getAttribute('data-story-url');
     this.element.classList.add('i-amphtml-story-dev-tools-tab');
+    return loadScript(this.element.ownerDocument, `${urls.cdn}/v0/validator.js`)
+      .then(() =>
+        this.validateUrl_(/* global amp: false */ amp.validator, this.storyUrl_)
+      )
+      .then((errorList) => {
+        this.errorList_ = errorList;
+        this.updateLogsTabIcon(errorList);
+      });
```

Comment:
Need to be done in the buildCallback in order to show the icon with the result. However, it doesn't create the list of errors in the tab content, just stores the errorList in `this.errorList_`

Reply:
Okay I see. As a general practice we don't do that in a buildCallback to never break privacy preserving prerendering. This page can't be prerendered though, but let's make it very obvious by adding a 

```
prerenderAllowed() {
  return false;
}
```